dlp/snippets/inspect/inspect_test.go

```diff
@@ -276,3 +276,21 @@ func TestInspectBigquery(t *testing.T) {
 		})
 	}
 }
+
+func TestInspectStringOmitOverlap(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := new(bytes.Buffer)
+
+	if err := inspectStringOmitOverlap(buf, tc.ProjectID, "gary@example.com"); err != nil {
+		t.Errorf("inspectStringOmitOverlap: %v", err)
```

Comment:
issue: replace `t.Errorf()` with `t.Fatal(err)`.

Reply:
fixed it.